### PR TITLE
Update NEWS.md before alpha11 release

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -20,7 +20,11 @@ OpenSSL 3.0
 
 ### Major changes between OpenSSL 1.1.1 and OpenSSL 3.0 [under development]
 
-  * Deprecated the `DSA_` functions.
+  * Deprecated the `OCSP_REQ_CTX` type and functions.
+  * Deprecated the `EC_KEY` and `EC_KEY_METHOD` types and functions.
+  * Deprecated the `RSA` and `RSA_METHOD` tyeps and functions.
+  * Deprecated the `DSA` and `DSA_METHOD` tyeps and functions.
+  * Deprecated the `DH` and `DH_METHOD` tyeps and functions.
```
Comment (t8m) [inline NEWS.md:27]: tyeps typos.

Comment (mattcaswell) [inline NEWS.md:27]: typo: tyeps (here and above lines)

Comment (levitte) [reply NEWS.md:27]: Fixed

**Review by t8m — APPROVED** [review]
LGTM now

**Review by mattcaswell — APPROVED** [review]

**Comment by levitte** [comment]
Merged

92bc61e467a2078438ce50ddda70a6afe6cf23df Update NEWS.md before alpha11 release

**Pull request closed without merging**